src/Symfony/Component/Workflow/Event/GuardEvent.php

```diff
@@ -27,9 +28,9 @@ class GuardEvent extends Event
     /**
      * {@inheritdoc}
      */
-    public function __construct($subject, Marking $marking, Transition $transition, $workflowName = null)
```

Comment:
The `$workflowName` => `WorkflowInterface $workflow = null` change should be done on a lower branch.